acceptance/tests/environment/custom_type_provider_from_same_environment.rb

```diff
@@ -62,6 +62,7 @@ def exists?
        '#{fq_tmp_environmentpath}/modules/simple_type/lib/puppet/type/':;
        '#{fq_tmp_environmentpath}/modules/simple_type/lib/puppet/provider/':;
        '#{fq_tmp_environmentpath}/modules/simple_type/lib/puppet/provider/#{type_name}':;
+       '#{fq_prod_environmentpath}/modules':;
```

Comment:
I think some other test might have left the environment in a weird way when it failed, which got us in this case where the base module path did not exist, causing nested ensures to fail.

As I fixed the other tests, this should also pass without the change, but I would leave this additional set up.